Unable to build with Docker

Hello,
first of all thanks for this project, it looks very promising!

I was trying to build the Dockerfile but the build fails due to problem with Ubuntu cosmic not receiving updates anymore (it's in EOL since July):

```
Step 5/22 : RUN apt-get update &&     apt-get install --no-install-suggests --no-install-recommends -y         $SYSTEM
 ---> Running in 0ef74c0e825b
Ign:1 http://security.ubuntu.com/ubuntu cosmic-security InRelease
Ign:2 http://archive.ubuntu.com/ubuntu cosmic InRelease
Err:3 http://security.ubuntu.com/ubuntu cosmic-security Release
  404  Not Found [IP: 91.189.88.149 80]
Ign:4 http://archive.ubuntu.com/ubuntu cosmic-updates InRelease
Ign:5 http://archive.ubuntu.com/ubuntu cosmic-backports InRelease
Err:6 http://archive.ubuntu.com/ubuntu cosmic Release
  404  Not Found [IP: 91.189.88.24 80]
Err:7 http://archive.ubuntu.com/ubuntu cosmic-updates Release
  404  Not Found [IP: 91.189.88.24 80]
Err:8 http://archive.ubuntu.com/ubuntu cosmic-backports Release
  404  Not Found [IP: 91.189.88.24 80]
Reading package lists...
E: The repository 'http://security.ubuntu.com/ubuntu cosmic-security Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu cosmic Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu cosmic-updates Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu cosmic-backports Release' does not have a Release file.
```

I've tried some luck by changing `FROM ubuntu:cosmic` to `FROM ubuntu:bionic` but seems like i had no luck:

```
Building wheels for collected packages: pyqtcolordialog, pluginbase, ffms2, lazy-import, parsimonious, ass-tag-parser, pympv, pyyaml
  Building wheel for pyqtcolordialog (setup.py): started
  Building wheel for pyqtcolordialog (setup.py): finished with status 'done'
  Created wheel for pyqtcolordialog: filename=pyqtcolordialog-0.3-py3-none-any.whl size=16296 sha256=2f20f2a7a8250f8abce0cb1f528526bfdfdb8f9f8ee3cf029aefa3eff00577d1
  Stored in directory: /root/.cache/pip/wheels/8c/ab/80/b9cfb37861668250c05a34681a803b810abe313be27dd6e3c2
  Building wheel for pluginbase (setup.py): started
  Building wheel for pluginbase (setup.py): finished with status 'done'
  Created wheel for pluginbase: filename=pluginbase-1.0.0-py3-none-any.whl size=7984 sha256=aa9218f9d18bc2c6bcc2845cde7395bdf8210ab55e1a1f1b33eb03165609bdca
  Stored in directory: /root/.cache/pip/wheels/63/76/be/ae588b56177703b847e362fbfec176c2fb8f84c811b5e41938
  Building wheel for ffms2 (setup.py): started
  Building wheel for ffms2 (setup.py): finished with status 'done'
  Created wheel for ffms2: filename=ffms2-0.4.5.2-py3-none-any.whl size=694527 sha256=061e8bd55b17ef574646c41f4e148e6328bc99f04a0eaba0d3efbe4d8f7c4b7c
  Stored in directory: /root/.cache/pip/wheels/36/f2/d5/707390ab7ecb7fa6645169da3069a4541c08d451718454dd19
  Building wheel for lazy-import (setup.py): started
  Building wheel for lazy-import (setup.py): finished with status 'done'
  Created wheel for lazy-import: filename=lazy_import-0.2.2-py2.py3-none-any.whl size=21025 sha256=9a425955f6e130e72707ad87769c4c64256cfccf4504b6626239a8de5eebae18
  Stored in directory: /root/.cache/pip/wheels/e6/8e/c7/c338956a635caa3b3153cd8e49b183badb75230ecf19144dff
  Building wheel for parsimonious (setup.py): started
  Building wheel for parsimonious (setup.py): finished with status 'done'
  Created wheel for parsimonious: filename=parsimonious-0.8.1-py3-none-any.whl size=50875 sha256=b941d08f0446fd42eae4326c9a641e5fa5b21bac500d67b42727ea705376205f
  Stored in directory: /root/.cache/pip/wheels/88/5d/ba/f27d8af07306b65ee44f9d3f9cadea1db749a421a6db8a99bf
  Building wheel for ass-tag-parser (setup.py): started
  Building wheel for ass-tag-parser (setup.py): finished with status 'done'
  Created wheel for ass-tag-parser: filename=ass_tag_parser-2.3-py3-none-any.whl size=23801 sha256=051a324349ec3d8205ec5b0bcfcd5b8161d3699eb6dce89164beca44e5ef6964
  Stored in directory: /root/.cache/pip/wheels/30/9e/e6/32b34e8146b4028729be7a91bb03b07fe6201988cd777d62cc
  Building wheel for pympv (setup.py): started
  Building wheel for pympv (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0hhw4rnw/pympv/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0hhw4rnw/pympv/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-yln7x2m1
       cwd: /tmp/pip-install-0hhw4rnw/pympv/
  Complete output (12 lines):
  running bdist_wheel
  running build
  running build_ext
  building 'mpv' extension
  creating build
  creating build/temp.linux-x86_64-3.7
  x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.7m -c mpv.c -o build/temp.linux-x86_64-3.7/mpv.o
  mpv.c:585:10: fatal error: mpv/render.h: No such file or directory
   #include "mpv/render.h"
            ^~~~~~~~~~~~~~
  compilation terminated.
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for pympv
  Running setup.py clean for pympv
  Building wheel for pyyaml (setup.py): started
  Building wheel for pyyaml (setup.py): finished with status 'done'
  Created wheel for pyyaml: filename=PyYAML-5.3-cp37-cp37m-linux_x86_64.whl size=45518 sha256=943486d0fbcb7cfe63daa345c5561073cbacf7f0d5f0d4408c3b98693635b4d1
  Stored in directory: /root/.cache/pip/wheels/8a/55/a4/c0a81d27c33462cfdcb904db018f5550197e88b2b6b85beed2
Successfully built pyqtcolordialog pluginbase ffms2 lazy-import parsimonious ass-tag-parser pyyaml
Failed to build pympv
Installing collected packages: PyQt5-sip, PyQT5, pyqtcolordialog, quamash, regex, Pillow, pluginbase, numpy, ffms2, lazy-import, parsimonious, ass-tag-parser, pyenchant, pympv, pyfftw, sortedcontainers, pyyaml, bubblesub
    Running setup.py install for pympv: started
    Running setup.py install for pympv: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0hhw4rnw/pympv/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0hhw4rnw/pympv/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-71coe82y/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.7/pympv
         cwd: /tmp/pip-install-0hhw4rnw/pympv/
    Complete output (12 lines):
    running install
    running build
    running build_ext
    building 'mpv' extension
    creating build
    creating build/temp.linux-x86_64-3.7
    x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.7m -c mpv.c -o build/temp.linux-x86_64-3.7/mpv.o
    mpv.c:585:10: fatal error: mpv/render.h: No such file or directory
     #include "mpv/render.h"
              ^~~~~~~~~~~~~~
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0hhw4rnw/pympv/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0hhw4rnw/pympv/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-71coe82y/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.7/pympv Check the logs for full command output.
```
I am not a python guy so i don't know where to start sorting out this issue.
Is it something you are already aware of? Do you have some advice to put me in the right direction?

Thanks!

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:cosmic
+FROM ubuntu:eoan
 
 MAINTAINER rr- "https://github.com/rr-"
 

